salt/cdn-logs/init.sls

```diff
@@ -14,6 +14,12 @@
   file.managed:
     - source: salt://cdn-logs/config/fastly.logrotate.conf
 
-/etc/cron.hourly/logrotate:
-  file.symlink:
-    - target: /etc/cron.daily/logrotate
+logrotate_time_hourly:
+  file.replace:
+    - name: /etc/systemd/system/timers.target.wants/logrotate.timer
+    - pattern: OnCalendar=daily
```

Comment:
My only concern with this approach is that if the value is changed to anything other than `daily` (manually, or via an update, or .... I dunno 👽), our state will not detect it and set the value we want `hourly`. This is probably OK, but a bit fragile.

Were you able to consider https://docs.saltproject.io/en/latest/ref/states/all/salt.states.ini_manage.html#salt.states.ini_manage.options_present for editing this file?